Library/Formula/mesos.rb

```diff
@@ -40,9 +88,35 @@ def install
 
     ENV.cxx11
 
-    system "./configure", *args
+    system "./configure", *(args + ["--disable-python"])
```

Comment:
Occurs I could also just pass the extra --en/disable-python argument in
front of *args; is that okay?